lib/standalone-tests/terminus.onmultiple.js

```diff
@@ -2,10 +2,10 @@
 const http = require('http')
 const server = http.createServer((req, res) => res.end('hello'))
 
-const { createTerminus } = require('../../')
+const { Terminus } = require('../../')
 const SIGNAL = process.argv[2]
 
-createTerminus(server, {
+const terminus = new Terminus(server, {
```

Comment:
@gergelyke standard is failing here because of "unused variable".. If I remove "const terminus =" it fails because of "no new" rule. What a dilemma..